installing/installing_ocp/install-secured-cluster-ocp.adoc

```diff
@@ -69,8 +67,11 @@ include::modules/change-config-options-after-deployment.adoc[leveloffset=+2]
 
 To install {product-title-short} on secured clusters by using the CLI, perform the following steps:
 
-. Install the `roxctl` CLI
-. Install Sensor.
+. Install the `roxctl` CLI if you will use the `roxctl` CLI command to generate the files used by the sensor installation script.
+. Generate the files used by the sensor installation script by performing one of the following actions:
+.. Log in to the {product-title-short} portal and download the cluster bundle zip archive.
+.. On the {ocp} secured cluster, in the terminal window, use the `roxctl sensor generate openshift` command.
```

Comment:
Are items 2a and 2b here a similar issue about alternatives as commented by Marcin elsewhere?
![roxctl_CLI](https://github.com/openshift/openshift-docs/assets/11862657/9a8ba6c4-a1b2-494e-aed8-a10cb43aab5b)

For my understanding as newbie to adoc, is `**` for open bullet at second level?

Reply:
Oops, I should've used `*` - thanks!